api/app/Listeners/StoreApplicationSnapshot.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\ApplicationSubmitted;
+use App\GraphQL\Util\GraphQLClient;
```

Comment:
Unused reference

Reply:
Removed in [remove unused import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3938/commits/c0effb3944d8e9ced544edd3d1210163823bd23e)